ntoskrnl/ke/i386/v86vdm.c

```diff
@@ -303,7 +311,13 @@ KiVdmOpcodeIRET(IN PKTRAP_FRAME TrapFrame,
     V86EFlags = EFlags;
 
     /* Check for VME support */
-    ASSERT(KeI386VirtualIntExtensions == FALSE);
+    if (KeI386VirtualIntExtensions)
+    {
+        if (EFlags & EFLAGS_INTERRUPT_MASK)
+        {
+            EFlags |= EFLAGS_VIF;
+        }
```

Comment:
KiVdmOpcodePOPF does something like
```
 144         /* Copy the IF flag into the VIF one */
 145         V86EFlags &= ~EFLAGS_VIF;
 146         if(V86EFlags & EFLAGS_INTERRUPT_MASK)
 147         {
 148             V86EFlags |= EFLAGS_VIF;
 149             /* Don't set the interrupt flag */
 150             V86EFlags &= ~EFLAGS_INTERRUPT_MASK;
 151         }
```
i.e. also removing EFLAGS_INTERRUPT_MASK. Just to be sure, here for this KiVdmOpcodeIRET handler we should keep the mask?